tests/integration/conftest.py

```diff
@@ -21,10 +40,61 @@ def pytest_collection_modifyitems(config, items):
             item.add_marker(skip_gpus)
 
 
-@pytest.fixture(scope="session")
-def modelstore(tmp_path_factory) -> "ModelStore":
+@pytest.fixture(scope="session", name="modelstore")
+def fixture_modelstore(tmp_path_factory: "TempPathFactory") -> "ModelStore":
     # we need to get consistent cache folder, thus tmpdir is not usable here
     # NOTE: after using modelstore, also use `delete_cache_model` to remove model after
     #  load tests.
     path = tmp_path_factory.mktemp("bentoml")
     return ModelStore(path)
+
+
+@pytest.fixture(scope="session", name="pipeline")
+def fixture_pipeline(modelstore: "ModelStore") -> Pipeline:
```

Comment:
this seems way too much complexity for something that can be done in a few lines of code, can we just use a utility function instead?

Reply:
I think in general fixtures are better fitted for initializing service/object and states. For a reusable "process", simple function calls are easier to manage